.github/workflows/ci_macos_arm64_clang.yml

```diff
@@ -23,7 +23,7 @@ concurrency:
 jobs:
   macos_arm64_clang:
     if: ${{ github.repository_owner == 'iree-org' || github.event_name != 'schedule' }}
-    runs-on: macos-14
+    runs-on: macos-15
```

Comment:
Can these be updated too?

https://github.com/iree-org/iree/blob/dfabc2b553390c360261b45c76bc4166e22e378d/.github/workflows/ci.yml#L73-L75 https://github.com/iree-org/iree/blob/dfabc2b553390c360261b45c76bc4166e22e378d/.github/workflows/ci.yml#L172

Any other references in the repo?

Reply:
Good catch. Updated all other references.